torchmdnet/datasets/ani.py

```diff
@@ -206,12 +208,12 @@ def get(self, idx):
 class ANI1(ANIBase):
     __doc__ = ANIBase.__doc__
     # Avoid sphinx from documenting this
-    ELEMENT_ENERGIES = {
```

Comment:
In line 49 there is this code `energy = sum(self._ELEMENT_ENERGIES[z] for z in atomic_numbers)`
which seems to assume that the classes have a `_ELEMENT_ENERGIES` variable. However here it's missing the underscore. Am I right in changing this?

Reply:
Definetely a bug. Thanks!